src/js/components/Calendar/Calendar.js

```diff
@@ -405,7 +405,7 @@ const Calendar = forwardRef(
               plain
               tabIndex={-1}
               active={active && active.getTime() === day.getTime()}
-              disabled={dayDisabled}
+              disabled={!!dayDisabled}
```

Comment:
```suggestion
              disabled={dayDisabled && !!dayDisabled}
```
I'm hoping this change will fix the changes on the snapshots. Let's give it a try.

Reply:
awe yes so it will only put `disabled={true}` on snapshots!